Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -1962,14 +1962,21 @@ final class RecoveryTests: XCTestCase {
       """
       // <rdar://problem/22387625> QoI: Common errors: 'let x= 5' and 'let x =5' could use Fix-its
       func r22387625() {
-        let _= 5
-        let _ =5
+        let _1️⃣= 4
+        let _ =2️⃣5
       }
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 3: '=' must have consistent whitespace on both sides, Fix-It replacements: 8 - 8 = ' '
-        // TODO: Old parser expected error on line 4: '=' must have consistent whitespace on both sides, Fix-It replacements: 10 - 10 = ' '
-      ]
+        DiagnosticSpec(locationMarker: "1️⃣", message: "'=' must have consistent whitespace on both sides"),
+        DiagnosticSpec(locationMarker: "2️⃣", message: "'=' must have consistent whitespace on both sides"),
```

Comment:
This position is wrong. We get 9 but expect 10. Not sure I'm finding the correct error position in the Cursor.swift

Reply:
Deleting derived data fixed it